RxSwiftExt.xcodeproj/xcshareddata/xcschemes/RxSwiftExtTests-iOS.xcscheme

```diff
@@ -5,6 +5,19 @@
    <BuildAction
       parallelizeBuildables = "YES"
       buildImplicitDependencies = "YES">
+      <BuildActionEntries>
```

Comment:
Any reason for the new Build Actions ?

Reply:
Hrm wondering about this one too...

Reply:
AppCode added some innocent scheme elements, sorry about that...

Reply:
Are they just innocent schemes ? I see build steps, kinda strange. 
We might want to add a .gitignore on xcshareddata so it doesn't track changes (e.g. the scheme stays static based on what's on the repo)